### ISSUE TITLE
I made an exact match GTK theme for people to use so they can have colored title bars.

### ISSUE THREAD

**Issue by freefreeno** [Mrcuve0/Aritim-Dark#2]
Not an issue but just letting you know I have made an exact match GTK theme for this. I will post it at kde store.

![Screenshot_20190602_150644](https://user-images.githubusercontent.com/41884680/58766797-937b9480-8548-11e9-9db4-838c1086b0d3.png)

![Screenshot_20190602_150721](https://user-images.githubusercontent.com/41884680/58766798-937b9480-8548-11e9-8819-f55e98397d05.png)

![Screenshot_20190602_150747](https://user-images.githubusercontent.com/41884680/58766799-937b9480-8548-11e9-9409-cfec12badfdc.jpg)

![Screenshot_20190602_150811](https://user-images.githubusercontent.com/41884680/58766800-937b9480-8548-11e9-8594-324cce80141a.png)

![Screenshot_20190602_150859](https://user-images.githubusercontent.com/41884680/58766802-94142b00-8548-11e9-9fed-62ee7a8e352c.png)

![Screenshot_20190602_153055](https://user-images.githubusercontent.com/41884680/58767275-6b8f2f80-854e-11e9-8930-bf2763830776.jpg)

**Comment by Mrcuve0** [comment]
Hi freefreeno! I don't know if you read the comment on the KDE Store that I sent some minutes ago.
I already made a GTK port of this theme and you can already find it on the KDE store (https://www.pling.com/p/1291666/).

You can also find it here among my repos.

**Comment by freefreeno** [comment]
I made mine from Breeze theme so I will rename it and post it. it is made for GTK in KDE only. Not for GTK in GNOME. Made especially for KDE. It has the colored title bars and all. I will name it Karitim-dark. It is posted as Karitim-dark-gtk.

**Comment by Mrcuve0** [comment]
As long as you are compliant with both the licenses there's no problem to me

**Comment by freefreeno** [comment]
It doesn't even have the same name so I don't see how it can not be compliant. If you use KDE you ought to check it out. There is not one single part that it does not match like to exact hex code.

**Comment by Mrcuve0** [comment]
I said that for you, to me there's no problem (as I said) as long as you are compliant with the GPLv3 license. Just to be "better safe than sorry". 

> There is not one single part that it does not match like to exact hex code.

As I'm not english native speaker I think I'm not properly understand what are you saying, can you rephrase this sentence? The double negation confuses me.

Does the HEX codes exactly match the ones used by the Aritim/Ayu theme or not?

**Comment by freefreeno** [comment]
Yes all hex codes match exactly but since it was made with the Breeze theme it also has the colored title bars. It will not work in Gnome. It is for use in KDE only with your theme.  I wasn't disagreeing with you. I was just saying that according to the license then it is all good.

**Comment by Mrcuve0** [comment]
Ok freefreeno, no problem, thank you for clarifying. I saw the product page: thank you for mentioning me.

Finally, if you liked the theme, feel free to leave a feedback!

Have a nice day!
